apps/artemis-web-e2e/src/integration/app.spec.ts

```diff
@@ -16,6 +16,8 @@ describe('artemis-web', () => {
     const name = chance.first();
     cy.typeRegister({ name: name, email: newEmail, password: newPass });
     cy.register();
+    cy.wait(1000);
+    cy.get('h1').should('have.text', 'Dashboard');
```

Comment:
I assume this is just for the e2e test to make sure that the correct page is shown? Or is it related to CSRF protection somehow? Asking because the changes are on this particular PR.

Reply:
you are right, this is  irrelevant to csrf. I was just passing the test while it was not working at some point